web/src/modules/layout/containers/LocationHeader.js

```diff
@@ -63,7 +61,7 @@ export class LocationHeader extends React.Component<PropsType> {
     ]
   }
 
-  getNavigationItems (): Array<Element<typeof HeaderNavigationItem>> {
+  getNavigationItems (): Array<React$Node> {
```

Comment:
Isn't the correct type React.Node here? I think this is the first time we use `React$Node`. Maybe I'm wrong :D

Reply:
Both is correct. `Reac$Node` can be used without importing anything. It is an official type though.
`/**
 * Copyright (c) 2013-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

/**
 * A UI node that can be rendered by React. React can render most primitives in
 * addition to elements and arrays of nodes.
 */
`

Reply:
Changed anyway for consistency